helix-query.yaml

```diff
@@ -0,0 +1,35 @@
+version: 1
+# See https://www.hlx.live/docs/setup-indexing.
+
+indices:
+  customer-success-stories:
+    include:
+      - '/**'
+    exclude:
+      - '**/fragments/**'
+    target: https://adobe.sharepoint.com/:x:/r/sites/adobecom/Shared%20Documents/bacom/customer-success-stories/query-index.xlsx
```

Comment:
Does this need the full sharepoint path?

Reply:
Nope